docs/shared_mappers_catalog/multipath_conf.rst

```diff
@@ -0,0 +1,3 @@
+.. automodule:: insights.mappers.multipath_conf
```

Comment:
This file needs to be moved to insights/docs/shared_parsers_catalog, and import needs to be insights.parsers.multipath_conf.